src/Slices/Settings/UI/Tabs/Configuration/Container.tsx

```diff
@@ -29,6 +43,19 @@ export const Container: React.FC<Props> = ({
           isInline
         />
       )}
+      {showUpdateBanner && (
+        <StyledAlert
+          variant="success"
+          title={"Setting Changed"}
```

Comment:
extract to the words list, I think that's also a standard we need to start doing. Atm it's not aligned and we have a mix. But everything that's printed to the screen, should be in that `words` list. That way, we have a central place for all hardcoded strings.